### DISCUSSION TITLE
Github Copilot is blocked in Jetbrains IDEA （version 2024.1.4）

### DISCUSSION THREAD

**Discussion by CxMYu** [community/community#129705] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I'm using github copilot in vs. No restrictions.
But when I use it in idea, it's blocked.
![image](https://github.com/community/community/assets/80197535/0fd9f008-4f0e-43ce-933f-d87cc1e74523)
![image](https://github.com/community/community/assets/80197535/ae875083-ad01-4a44-8cd6-d20bb07a683b)

**Comment by Akash1134** [comment, marked as answer]
Hi @CxMYu ,

Please check whether your organization/school employs security measure like proxy, it may be beneficial to include certain domain [URLs in an "allowlist" ](https://docs.github.com/en/enterprise-cloud@latest/copilot/troubleshooting-github-copilot/troubleshooting-firewall-settings-for-github-copilot) I would like to check if you have allowed these URLs on your proxy/firewall. 

> [!Note]
> default.exp-tan.com is one of the telemetry URL's we suggest marking as trusted either in the user interface or within your deployment scripts. 

<img width="784" alt="Screenshot 2024-06-25 at 2 33 27 PM" src="https://github.com/user-attachments/assets/b9ae64ab-2a1d-4447-952f-f6a1db35a74c">

**Reply by CxMYu** [reply]
I think I'm on to something.
But how come it doesn't work in IDEA but it's supported in Visual studio 2022.
